### DISCUSSION TITLE
Rename Alert Field

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#8092] [Unsupported Versions] [answered]
Is it possible to rename the field titles (rule.name) = Rule  or event.severity_label to Severity?   
I thought it might be handled in the alert.eventfields.json, but moving to local and changing didnt do anything.

Thanks

**Comment by dougburks** [comment, marked as answer]
No, the fields are shown exactly as they exist in Elasticsearch.
